h/static/styles/share-link.scss

```diff
@@ -27,6 +33,30 @@
   width: 100%;
 }
 
+.share-target {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  border-bottom: 1px solid $gray-lighter;
+  font-size: $body1-font-size;
+  padding: 10px;
+}
+
+.share-target__label {
+  font-weight: bold;
+}
+
+.share-target__icon {
+  color: #7A7A7A;
```

Comment:
Are these colors from the new design palette? If so I'd suggest that we introduce variables for them in `variables.scss`, using Conor's naming for them.